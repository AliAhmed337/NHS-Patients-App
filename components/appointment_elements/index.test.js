import React from 'react';
import { Text } from 'react-native';
import { shallow } from 'enzyme';
import { AppointmentBody,
		 AppointmentContainer,
		 AppointmentDetail,
		 AppointmentHeader,
		 AppointmentTimer,
		 AppointmentTitle
		} from './index';

describe('Testing appointment components', () => {
	it('renders correctly', () => {
		const bodyWrapper = shallow(
			<AppointmentBody>
				<Text>body</Text>
			</AppointmentBody>
		);
		expect(bodyWrapper).toMatchSnapshot();

		const containerWrapper = shallow(
			<AppointmentContainer>
				<Text>container</Text>
			</AppointmentContainer>
		);
		expect(containerWrapper).toMatchSnapshot();

		const detailWrapper = shallow(
			<AppointmentDetail>
				<Text>detail</Text>
			</AppointmentDetail>
		);
		expect(detailWrapper).toMatchSnapshot();
		
		const headerWrapper = shallow(
			<AppointmentHeader>
				<Text>header</Text>
			</AppointmentHeader>
		);
		expect(headerWrapper).toMatchSnapshot();

		const timerWrapper = shallow(
			<AppointmentTimer>
				<Text>timer</Text>
			</AppointmentTimer>
		);
		expect(timerWrapper).toMatchSnapshot();

		const titleWrapper = shallow(
			<AppointmentTitle>
				<Text>title</Text>
			</AppointmentTitle>
		);
		expect(titleWrapper).toMatchSnapshot();
	});
});