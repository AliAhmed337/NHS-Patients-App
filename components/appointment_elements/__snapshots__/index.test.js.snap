// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing appointment components renders correctly 1`] = `ShallowWrapper {}`;

exports[`Testing appointment components renders correctly 2`] = `ShallowWrapper {}`;

exports[`Testing appointment components renders correctly 3`] = `ShallowWrapper {}`;

exports[`Testing appointment components renders correctly 4`] = `ShallowWrapper {}`;

exports[`Testing appointment components renders correctly 5`] = `ShallowWrapper {}`;

exports[`Testing appointment components renders correctly 6`] = `ShallowWrapper {}`;
